pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -232,6 +242,156 @@ func getMachineDefaulterOperation(platformStatus *osconfigv1.PlatformStatus) mac
 	}
 }
 
+// NewValidatingWebhookConfiguration creates empty validation webhook configuration
+func NewValidatingWebhookConfiguration() *admissionregistrationv1.ValidatingWebhookConfiguration {
```

Comment:
why is this split from `admissionregistrationv1.ValidatingWebhookConfiguration`?
Can just the full spec be defined here?

Reply:
Ok. I just wanted to add more visibility to what is done there and make unit testing simpler. Right now unit tests Joel made are accounting for both validating and mutating webhooks, so to cover it all we need selectively disable one of them.